### PR TITLE
fix:Error in fetching notes.txt for helm charts deployed via gitops  when resource is present previously.

### PR DESCRIPTION
**Pull request by adi6859**
# Description
This  PR aims to fix notes.txt fetching error (as shown below) when a user tries to fetch notes for a helm chart.
Screenshot 2023-03-20 at 3 55 11 PM

Fixes [AB38](https://github.com/devtron-labs/kubelink/issues/38)
## How Has This Been Tested?
1. I have done testing this via image deployment on demo.
2. I have checked for all possible test cases.
Test cases are:-
a.deployed  helm chart via gitOps when no apps is present.
b.deleted above helm app and again installed same chart with same name.
c. deployed  same helm chart again when above app  is present.
4. I have also done testing by applying multiple breakpoint for checking data is coming or not.
5. i have also checked via main.go file by calling my function.

## Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code.
* [x] I have commented my code, particularly in hard-to-understand areas.
* [ ] I have tested it for all user roles.
* [ ] I have added all the required unit/api test cases.

### CODE REVIEW

File: pkg/service/HelmAppService.go [hunk 1]
```diff
@@ -545,14 +545,28 @@ func (impl HelmAppServiceImpl) installRelease(request *client.InstallReleaseRequ
...
+	}
+	HelmTemplateOptions := &helmClient.HelmTemplateOptions{}
+	release, err := helmClientObj.GetNotes(chartSpec, HelmTemplateOptions)
 	if err != nil {
 		impl.logger.Errorw("Error in fetching Notes ", "err", err)
 		return "", err
 	}
-	return release.Info.Notes, nil
+	if release == nil {
+		return "", errors.New("release is found nil")
```
Comment (vikramdevtron) [inline pkg/service/HelmAppService.go:566]: add logger, put request there, change error message in response "no release found with the name = release name"

File: pkg/service/HelmAppService.go [hunk 2]
```diff
@@ -545,14 +545,28 @@ func (impl HelmAppServiceImpl) installRelease(request *client.InstallReleaseRequ
...
+	}
+
+	return string(release), nil
 }
```
Comment (vikramdevtron) [inline pkg/service/HelmAppService.go:570]: add unit test case for this function

Comment (adi6859) [reply pkg/service/HelmAppService.go:570]: done

File: pkg/service/HelmAppService.go (hunk 1 above)
Comment (adi6859) [reply pkg/service/HelmAppService.go:566]: done

**Review by vikramdevtron — APPROVED** [review]

**Pull request merged**